dashboard/lib/sign_up_tracking.rb

```diff
@@ -71,6 +79,14 @@ def self.log_begin_sign_up(user, session)
       }.to_json
     }
     FirehoseClient.instance.put_record(:analysis, tracking_data)
+
+    Metrics::Events.log_event(
+      event_name: "#{provider}-begin-sign-up-#{result}",
```

Comment:
Potentially dumb question (might be a brain glitch), where is `provider` defined in the scope of this method? I see it's a a parameter in the methods below, but not sure it's defined here?

Reply:
You're right, I somehow missed this and removed it. Thank you!